tests/karma/unit/services/db.js

```diff
@@ -17,6 +17,7 @@ describe('DB service', function() {
     module(function ($provide) {
       $provide.factory('$window', function() {
         return {
+          angular: { callbacks: [] },
```

Comment:
IDK. It's an unrelated issue that I've cherry-picked to master too. I think angular must've updated something internally so mocking $window is even harder  :(